x/examples/fyne-tools/README.md

```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
 Note: the generated APK is around 85MB.
 
 
+## Windows
```

Comment:
Maybe add "cross-compilation from macOS" to the title? Cuz Windows users can build it directly from Windows (might still need to install either `gcc` or `msvc`).

```suggestion
## Windows (x-compile)
```

Reply:
Updated the text.